starport/interface/cli/starport/cmd/network_chain_join.go

```diff
@@ -96,7 +96,7 @@ func networkChainJoinHandler(cmd *cobra.Command, args []string) error {
 	if err := cliquiz.Ask(questions...); err != nil {
 		return err
 	}
-	gentx, address, mnemonic, err := blockchain.IssueGentx(cmd.Context(), account, proposal)
+	gentx, a, err := blockchain.IssueGentx(cmd.Context(), account, proposal)
```

Comment:
Small detail but I think it's better to keep explicit name for variables

Reply:
`account` and `acc` are already in use here. I had to use something else. but for sure we can improve this on the next iterations.